pkg/cmd/pr/checks/checks.go

```diff
@@ -132,6 +132,20 @@ func checksRun(opts *ChecksOptions) error {
 		return clientErr
 	}
 
+	var checks []check
+	var counts checkCounts
+	var err error
+
+	err = populateStatusChecks(client, repo, pr)
+	if err != nil {
+		return err
+	}
+
+	checks, counts, err = aggregateChecks(pr, opts.Required)
```

Comment:
Since we now do the combo of `populateStatusChecks` + `aggregateChecks` twice, would it make sense to capture both actions as a single helper function?